### PR TITLE
Added xio-0.13.0 as a dependency

### PR DESCRIPTION
**Pull request by pdex**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -50,13 +51,15 @@ tasks.withType(JavaCompile) {
 
 repositories {
     mavenCentral()
+    maven { url 'https://dl.bintray.com/nordstromoss/oss_maven' }
```
Comment (andyday) [inline build.gradle]: using jcenter() as repo was sufficient for apim-api to get xrpc from bintray.  not sure why we would need any different here.

Comment (jkinkead) [reply build.gradle]: This project isn't synced to jcenter (yet). I assume @pdex has opened a ticket to get it syncing?

Comment (pdex) [reply build.gradle]: yes!

Comment (pdex) [reply build.gradle]: jcenter has approved my request, but xio hasn't landed there yet.

**Comment by andyday** [comment]
excellent!  👍

**Pull request merged**